packages/withoutHTML/src/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @description Cleanups a string from HTML tags.
+ * @param {String} text
+ * @example
+ * withoutDiacritics('<b>Test</b>') // returns 'Test'
+ */
+
+function removeHTML(ogstr)
+{
+   if ((ogstr===null) || (ogstr===''))
+       return false;
+  else
+   ogstr = ogstr.toString();
+  return ogstr.replace(/<[^>]*>/g, '');
+}
+ 
+const withoutHTML = text => removeHTML(text);
```

Comment:
Can you please create an inline function? 
```
const withoutHTML = text => {}
```

Reply:
Done, hope it's good.